### ISSUE TITLE
Hiding arrows has no purpose, how to change slides?

### ISSUE THREAD

**Issue by LMS007** [femioladeji/react-slideshow#36]
There should be a way to programmably change to a slide via some sort of id. I'm trying to do this with refs, but hard to follow compiled code with no examples.

I'm just doing this for now to change to second slide, and it feels pretty hacky...
```

<button onClick={()=>{
        this.slides.current.goToSlide({
          target: {
            dataset: {
              key: 1
            }
          }
        });
      }}>test</button>
      <Slide {...properties} ref={this.slides} >
          ...
```

**Comment by femioladeji** [comment]
Hi @LMS007 if I get you right, you want to programmatically navigate to a particular slide image without clicking on the arrows or the indicators just below the slide. Is that what you want?

**Comment by LMS007** [comment]
Yes, this is correct. Might also be nice to have a callback when the slide animation finishes. Purpose for that is to set css classes to kick off key frame animations... nice to have!

Thank you

**Comment by femioladeji** [comment]
Ok I'll work on that. Thanks for pointing it out @LMS007

**Comment by femioladeji** [comment]
#37

**Comment by LMS007** [comment]
> PR raised to fix this #37. Thanks for your input @LMS007

@femioladeji Great news, thank you!
